rls/deploy/deploy_cid_rls.yaml

```diff
@@ -0,0 +1,210 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Lambda to collect Org data and store in S3 
+Parameters:
+  DestinationBucket:
+    Type: String
+    Description: Name of the S3 Bucket that is created to hold org data
+    AllowedPattern: (?=^.{3,63}$)(?!^(\d+\.)+\d+$)(^(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])$)
+  ManagementAccountRole:
+    Type: String
+    Description: The name of the IAM role that will be deployed in the management account which can retrieve AWS Organization data. KEEP THE SAME AS WHAT IS DEPLOYED INTO MANAGEMENT ACCOUNT
+    Default: WA-Lambda-Assume-Role-Management-Account
```

Comment:
Please note that this role deployed by DC stack in Mngmnt acc has now restrictions on the Principal ARN. This will not work for just any lambda role 

https://github.com/awslabs/cid-framework/blob/main/data-collection/deploy/deploy-in-management-account.yaml#L108

Reply:
Also default prefix has changed

Reply:
addressed in https://github.com/awslabs/cid-framework/pull/153/commits/ddb53826490b95ed436732a2143cff2b290bae37